cron/run_benchmarks.sh

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/bash
+echo "Running benchmark update `date`"
+BENCHMARK_REPO=${BENCHMARK_REPOSITORY:-$HOME/dask-benchmarks}
+DASK_DIR=$BENCHMARK_REPO/dask
+DISTRIBUTED_DIR=$BENCHMARK_REPO/distributed
+DASK_CONFIG=${DASK_ASV_CONFIG:-$HOME/asv.dask.conf.json}
+DISTRIBUTED_CONFIG=${DISTRIBUTED_ASV_CONFIG:-$HOME/asv.distributed.conf.json}
+
+source activate dask-asv
+
+echo "Updating benchmark repo..."
+cd $BENCHMARK_REPO
+git checkout master
+git pull
+
+echo "Running dask benchmarks..."
+cd $DASK_DIR
+asv --config $DASK_CONFIG run NEW
```

Comment:
Should also run new benchmarks, see https://github.com/spacetelescope/asv/issues/278